evap/staff/views.py

```diff
@@ -796,6 +797,39 @@ def semester_participation_export(_request, semester_id):
     return response
 
 
+@manager_required
+def vote_timestamps_export(_request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+    timestamps = VoteTimestamp.objects.filter(evaluation__course__semester=semester)
```

Comment:
Although this is not really performance critical, we could prefetch the fields we use below here